File: mercury-0.13.1-r2/comment.m
:- module comment.
:- interface.

:- import_module io.
:- pred main( io.state, io.state).

:- implementation.

:- import_module string.
:- import_module testenv.

:- pred runtest( testdata, testdata, io.state, io.state).
:- mode runtest( in, out, di, uo) is det.
runtest( !TD, !IO) :-
 checkComplainAndAdjustExpected( !TD, 0, !IO),

 % this is a line comment

 /*
  this is a block comment
 */

 checkComplainAndAdjustExpected( !TD, 0, !IO),
 true
.

:- mode main( di, uo) is det.
main( !IO) :- 
 runtest( testdata( 0, 0, 0), TD, !IO),
 io.write_string( result( TD)++"\n", !IO),
 true
.

:- end_module comment.

File: swi-prolog-5.6.62/interpreter/comment.pl

main :- 

 consult( testenv),

 checkComplainAndAdjustExpected( 0),

 % this is a line comment

 /*
  this is a block comment
 */

 checkComplainAndAdjustExpected( 0),

 % see also : 
 
 result(S), 
 format( S, []),

 true.

